aspnetcore/grpc/browser.md

```diff
@@ -75,6 +75,26 @@ Traditional gRPC over HTTP/2 supports streaming in all directions. gRPC-Web offe
 
 When using gRPC-Web, we only recommend the use of unary methods and server streaming methods.
 
+### HTTP protocol
+
+The ASP.NET Core gRPC service template included with .NET creates an app that's configured for HTTP/2 only. This is a good default when an app only supports traditional gRPC over HTTP/2. gRPC-Web, however, works with both HTTP/1.1 and HTTP/2. Some platforms, such as UWP or Unity, can't use HTTP/2. To support all client apps, configure the server to enable HTTP/1.1 and HTTP/2.
+
+The default protocol can be updated in `appsettings.json`:
```

Comment:
Imperative tone will eliminate the state verb (be) ...

```suggestion
Update the default protocol in `appsettings.json`:
```

If you prefer to give this approach and the Kestrel endpoint config approach more equal weight, a lead in sentence and list is an option.